frontend/app/src/locales/en.json

```diff
@@ -390,6 +390,33 @@
     "no_messages": "No messages!",
     "title": "Notifications"
   },
+  "oracle_cache_management": {
+    "clear_error": "Clearing the case for {fromAsset} to {toAsset} failed: {error}",
+    "create_cache": "Cache Pair prices",
+    "notification": {
+      "title":  "Price pair cache creation",
+      "error": "The pair {fromAsset} to {toAsset} failed: {error}",
```

Comment:
Here and below:

The cache creation for pair ...


Also if possible include the oracle name here.